virttest/utils_misc.py

```diff
@@ -4559,3 +4560,18 @@ def get_pid(name, session=None):
         return None
     else:
         return int(output.split()[1])
+
+
+def start_rsyslogd():
+    """
+    Start rsyslogd service
+    """
+    try:
+        utils_path.find_command("rsyslogd")
+    except utils_path.CmdNotFoundError:
+        exceptions.TestError("No rsyslogd command found.")
+    rsyslogd = service.Factory.create_service('rsyslog')
```

Comment:
Do you need first check whether one rsyslogd service is already running before you create one?

Reply:
the check is taken care in the next line.